ethpm_cli/registry.py

```diff
@@ -25,8 +26,34 @@ def format_for_display(self) -> str:
         return f"{self.uri} --- {self.alias}{activated}"
 
 
+def deploy_registry(config: Config, alias: str = None) -> str:
+    if not config.private_key:
+        raise AuthorizationError(
+            "To deploy a registry, you must provide the password for your local keyfile."
+        )
+    chain_id = config.w3.eth.chainId
+    chain_name = SUPPORTED_CHAIN_IDS[chain_id]
+    cli_logger.info(
+        f"Deploying a new registry to {chain_name}, this may take a minute..."
+    )
+    # todo: handle tx timeout error gracefully
+    registry_address = config.w3.pm.deploy_and_set_registry()
```

Comment:
Not needed for this PR but the UX for this would be significantly improved if it gave continuous feedback as the various things happened.

```
Deploying new registry to ....
Broadcast transaction 0x1234... deploying registry contract....
Waiting for tx to be mined....
Transaction included in block #81234234
Doing other thing....
...
```